assets/src/containers/App.js

```diff
@@ -1,4 +1,3 @@
-/* global myla_globals */
```

Comment:
This comment was for Codacy but I guess it didn't need it anymore with the change of variable.

Reply:
Yeah, because I opted to use the `json_script` tag and the `document.getElementById` call, I don't think we need it anymore?